.github/workflows/donotmerge.yaml

```diff
@@ -0,0 +1,23 @@
+name: Do Not Merge
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - labeled
+    - unlabeled
+    branches:
+    - '*'
+
+jobs:
+  do-not-merge:
+    if: ${{ contains(github.event.*.labels.*.name, 'do-not-merge/hold') }}
```

Comment:
quick confirmation, does this label be set/unset by reviewer as author can always make PR to draft status and thus preventing the merge?

Reply:
ack. let's see the usage and re-visit then. good for now.

Reply:
btw, I don't have the option to resolve the conversion, only the author and repo owner will have that option afaik 🤔

Reply:
@leelavg The ability to hold a PR is important when there is external context (not part of the PR itself) that should hold the PR from being merged. That context might not be available to a specific reviewer or approver.

Reply:
- Yes, I already agreed with Dhruv's explanation.
- What I'm saying is, since I raised the comment and I need to resolve it I'm not able to see the option to resolve conversation 😕

Reply:
Even I'm not able to see the resolve conversation option on other PR's that I reviewed.

Reply:
Resolving this thread for now as Leela is unable to resolve